src/components/Nav.svelte

```diff
@@ -32,7 +32,8 @@
 				{ txt: 'Companions',  link: 'companions' },
 				{ txt: 'Battle Pets', link: 'battlepets' },
 				{ txt: 'Toys',        link: 'toys'       },
-				{ txt: 'Titles',	  link: 'titles'     }
+				{ txt: 'Titles',	  link: 'titles'     },
+				{ txt: 'Heirlooms',	  link: 'heirlooms'  },
```

Comment:
Heirlooms feels more like a collectable than titles, imo we can put heirlooms above titles in the menu